runtime/js/98_global_scope.js

```diff
@@ -17,118 +16,118 @@ import * as url from "ext:deno_url/00_url.js";
 import * as urlPattern from "ext:deno_url/01_urlpattern.js";
 import * as headers from "ext:deno_fetch/20_headers.js";
 import * as streams from "ext:deno_web/06_streams.js";
-// Unused import, required to work around Deno's check:
-// "Following modules were not evaluated; make sure they are imported from other code"
-import * as fileReader from "ext:deno_web/10_filereader.js";
+// import * as fileReader from "ext:deno_web/10_filereader.js";
```

Comment:
Are these unusded imports still required as a workaround? If so let's add a comment above them like we use to have (see the diff).

Reply:
Oh okay, got it!